ui/src/app/views/homepage/component.tsx

```diff
@@ -186,153 +160,125 @@ export class WorkbenchAccessTasks extends
   }
 }
 
-@Component({
-  selector: 'app-workbench-access-tasks',
-  template: '<div #root></div>',
-})
-export class WorkbenchAccessTasksComponent extends ReactWrapperBase {
-  @Input('eraCommonsLinked') eraCommonsLinked: WorkbenchAccessTasksProps['eraCommonsLinked'];
-  @Input('eraCommonsError') eraCommonsError: WorkbenchAccessTasksProps['eraCommonsError'];
-  @Input('trainingCompleted') trainingCompleted: WorkbenchAccessTasksProps['trainingCompleted'];
-  constructor() {
-    super(WorkbenchAccessTasks, ['eraCommonsLinked', 'eraCommonsError', 'trainingCompleted']);
+const homepageStyles = reactStyles({
+  backgroundImage: {
+    backgroundImage: 'url("/assets/images/AoU-HP-background.jpg")',
+    backgroundRepeat: 'no-repeat', backgroundSize: 'cover', height: '100%',
+    marginLeft: '-1rem', marginRight: '-0.6rem', display: 'flex',
+    flexDirection: 'column', justifyContent: 'space-between'
+  },
+  singleCard: {
+    height: '33.4%', width: '87.34%', minHeight: '18rem', maxHeight: '26rem',
+    display: 'flex', flexDirection: 'column', borderRadius: '5px',
+    backgroundColor: 'rgba(255, 255, 255, 0.15)',
+    boxShadow: '0 0 2px 0 rgba(0, 0, 0, 0.12), 0 3px 2px 0 rgba(0, 0, 0, 0.12)',
+    border: 'none', marginTop: '1rem'
+  },
+  quickRow: {
+    display: 'flex', justifyContent: 'flex-start', maxHeight: '26rem',
+    flexDirection: 'row', marginLeft: '4rem', padding: '1rem'
+  },
+  quickTourLabel: {
+    fontSize: 28, lineHeight: '34px', color: '#fff', paddingRight: '2.3rem',
+    marginTop: '2rem', width: '33%'
+  },
+  footer: {
+    height: '300px', width: '100%', backgroundColor: '#262262',
+    boxShadow: '0 0 2px 0 rgba(0, 0, 0, 0.12), 0 3px 2px 0 rgba(0, 0, 0, 0.12)',
+    marginTop: '2%',
+  },
+  footerInner: {
+    display: 'flex', flexDirection: 'column', marginLeft: '5%', marginRight: '5%',
+  },
+  footerTitle: {
+    height: '34px', opacity: 0.87, color: '#fff', fontSize: 28,
+    fontWeight: 600, lineHeight: '34px', width: '87.34%', marginTop: '1.4rem'
+  },
+  footerText: {
+    height: '176px', opacity: 0.87, color: '#83C3EC', fontSize: '16px',
+    fontWeight: 400, lineHeight: '30px', display: 'flex', width: '100%',
+    flexDirection: 'column', flexWrap: 'nowrap', overflowY: 'scroll'
+  },
+  linksBlock: {
+    display: 'flex', marginBottom: '1.4rem', marginLeft: '1.4rem',
+    flexDirection: 'column', flexShrink: 1, minWidth: 0
+  },
+  bottomBanner: {
+    borderTopColor: '#5DAEE1', borderStyle: 'solid', borderWidth: '0.3rem 0 0 0',
+    width: '100%', display: 'flex', backgroundColor: '#483F4B', height: '5rem',
+  },
+  logo: {
+    top: '1rem', left: '6.25rem', height: '3.5rem', width: '7rem',
+    position: 'relative', lineHeight: '85px'
+  },
+  bottomLinks: {
+    color: '#9B9B9B', fontSize: '0.7rem', height: '1rem', left: '5.5rem',
+    top: '2rem', marginLeft: '2.5rem', position: 'relative', fontWeight: 400
+  },
+  addCard: {
```

Comment:
Question: do you think it should be moved to components/card.tsx as i can see this being used at other places as well or is it specific only to homepage?

Reply:
so there already is a `CardButton` element that has some default styles (in buttons.tsx).  i think some of the styles are custom to this specific card (where it sits on the homepage) but some could be moved out commonly.  In that case though I think we'd want it to be an "AddCardButton" built off of `CardButton` since the color/text size/plus button icon are all specific to the add function (we see them on the list pages).  I will make that change now.

Reply:
scratch that, i think the common styles should be in `CardButton` itself